meeting_minutes/221013_SARIF_TC_68.md

```diff
@@ -0,0 +1,154 @@
+  * Michael continues the action
+* ACTION on Mary to continue to develop fixes scenario
+  * ONGOING
+* ACTION on Michael to provide additional references for the wikipedia
+  * COMPLETE
+* ACTION on Jeff to provide an initial seed page at github to foster the future roadmap / extension discussion
+  * ONGOING
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Charles Wilson
+* Paul Anderson
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair Keaton)
+
+- Scheduled Teleconferences (Thursdays at 08:00 PT / 15:00 UTC for 1.5 hours)
+    ```
+    October 27
+    ```
+- Possible face-to-face meeting when pandemic permits
+
+# 3. Liaisons
+
+## 3.1 [OpenSSF](https://openssf.org/) Security Tools group
+
+> Liaison Group: ForAllSecure/Thanassis Avgerinos, Contrast Security/Jeff Williams, Microsoft/Michael Fanning
+
+* Report status and any communication from OpenSSF Security Tools
+  * Thanassis: no update as meeting was cancelled
+* Compose a liaison statement to OpenSSF Security Tools if appropriate
+  * No update
+
+# 4. Discussion
+
+## 4.1 Review status on finalizing SARIF 2.1 errata [List of next steps - github issue #509](https://github.com/oasis-tcs/sarif-spec/issues/509)
+
+* David: Addressed all comments from Yekatarina and all but one comment from Michael. Plans to move to Michael after final comment addressed.
+* Michael: Courtneys departure requires maybe some additional arrangements for support.
+
+## 4.2 Review current state of ecosystem ongoing work
+
+* Mary: Reports on big Microsoft push towards security and thus pushes the SARIF (press announcement Yesterday) w.r.t. Github advanced security and Azure.
+* Mary: C++ compiler team more closely working with the Visual Studio itself
+* Mary: Also plans to include the SARIF functionality included into the Visual Studio installer.
+
+## 4.3 Review outcomes of subgroup discussions
+
+* No news.
+
+## 4.4 Discuss end-to-end results management
+
+* No news.
+
+## 4.5 Review Roadmap
+
+* Michael: 7 topics identified within TC
+  1. Close Wikipedia page and errata
+  1. Review non-breaking changes for an incremental release
+  1. SARIF eco-system advances that serve TC members / others (fixes)
+  1. Reference implementations for fingerprints, REST API, SARIF-as-SQL, etc.
+  1. Exploring deep topics at the "edges" of SARIF, e.g. results management
+  1. Scaling (large files), signing, and encryption
+  1. Broad advances into new topics, e.g. dynamic analysis
+* Michael: 1 or more topics possibly outside of the TC
+  1. New initiatives
```

Comment:
should these be numbered 1 through 8?

Reply:
Every markdown processor I use does that, but the reason I use the same number is, that in case you reorder, the diff is less noisy.